README.md

```diff
@@ -29,215 +29,21 @@ in alpha, and other platforms planned.
-encountered.
-
-
-## Installing
-
-### Prerequisite
-
-`kubectl` is required, see [here](https://kubernetes.io/docs/tasks/tools/install-kubectl/).
-
-
-### OSX From Homebrew
-
-```console
-brew update && brew install kops
-```
-
-The `kops` binary is also available via our [releases](https://github.com/kubernetes/kops/releases/latest).
-
-
-### Linux
-
-```console
-curl -LO https://github.com/kubernetes/kops/releases/download/$(curl -s https://api.github.com/repos/kubernetes/kops/releases/latest | grep tag_name | cut -d '"' -f 4)/kops-linux-amd64
-chmod +x kops-linux-amd64
-sudo mv kops-linux-amd64 /usr/local/bin/kops
-```
-
-### Windows
-
-1) Get `kops-windows-amd64` from our [releases](https://github.com/kubernetes/kops/releases/latest).
-2) Rename `kops-windows-amd64` to `kops.exe` and store it in a preferred path.
-3) Make sure the path you chose is added to your `Path` environment variable.
+## Releases and kubernetes Release Compatibility
 
-## Release History
-
-See the [releases](https://github.com/kubernetes/kops/releases) for more
-information on changes between releases.
+See [Releases and versioning](https://kops.sigs.k8s.io/welcome/releases/)
 
 
 ## Getting Involved and Contributing
 
-Are you interested in contributing to kops? We, the maintainers and community,
-would love your suggestions, contributions, and help! We have a quick-start
-guide on [adding a feature](/docs/development/adding_a_feature.md). Also, the
-maintainers can be contacted at any time to learn more about how to get
-involved.
-
-In the interest of getting more newer folks involved with kops, we are starting to
-tag issues with `good-starter-issue`. These are typically issues that have
-smaller scope but are good ways to start to get acquainted with the codebase.
-
-We also encourage ALL active community participants to act as if they are
-maintainers, even if you don't have "official" write permissions. This is a
-community effort, we are here to serve the Kubernetes community. If you have an
-active interest and you want to get involved, you have real power! Don't assume
-that the only people who can get things done around here are the "maintainers".
-
-We also would love to add more "official" maintainers, so show us what you can
-do!
-
-
-What this means:
-
-__Issues__
-* Help read and triage issues, assist when possible.
-* Point out issues that are duplicates, out of date, etc.
-  - Even if you don't have tagging permissions, make a note and tag maintainers (`/close`,`/dupe #127`).
-
-__Pull Requests__
-* Read and review the code. Leave comments, questions, and critiques (`/lgtm` ).
-* Download, compile, and run the code and make sure the tests pass (make test).
-  - Also verify that the new feature seems sane, follows best architectural patterns, and includes tests.
-
-This repository uses the Kubernetes bots.  See a full list of the commands [here](
-https://go.k8s.io/bot-commands).
-
-
-## Office Hours
```

Comment:
I do think we should have a link to office hours on the "homepage"

Reply:
I added a shorter description of office hours now, with a link to the longer one.

Reply:
Thanks for adding them, I agree it's good to have the link.